### DISCUSSION TITLE
Show variable in Cycle View Mode

### DISCUSSION THREAD

**Discussion by deejayluk** [grafana/grafana#38168] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
Please add option to shows variable when dashboard is on the cycle view mode.
**Why is this needed**:
I am using Grafana to visualizate dashboards on the production and for me this is very important options.
Variables for me are filters, users can choose some options who have effect on dashboards.

**Comment by lluviameneses** [comment]
Agree...it would be a very useful feature. Thanks

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by sshuair** [comment]
any update?

**Comment by erturkmemmedli** [comment]
any update?

**Comment by alexandreamormino95** [comment]
any update?

**Comment by NWYLZW** [comment]
any update?

**Comment by liuhong110** [comment]
any update?

**Comment by NWYLZW** [comment]
Now, you can look [there](https://github.com/grafana/grafana/issues/20176#issuecomment-1610959338)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
